### PR TITLE
[IMP] hr_holidays: add various cosmetic changes

### PR DESCRIPTION
**Pull request by Aurelienvd**
Remove 'refuse' leaves from the Calendar View.
Improve leave/leave.allocation popup form views.
Add Sequence handle to leave type list view.

Task ID: 2377043

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/security/hr_holidays_security.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
     <record id="hr_leave_rule_employee_unlink" model="ir.rule">
         <field name="name">Time Off base.group_user unlink</field>
         <field name="model_id" ref="model_hr_leave"/>
-        <field name="domain_force">[('employee_id.user_id', '=', user.id), ('state', '=', 'draft')]</field>
+        <field name="domain_force">[('employee_id.user_id', '=', user.id), ('state', 'not in', ['vaidate', 'validate1'])]</field>
```
Comment (kba-odoo) [inline addons/hr_holidays/security/hr_holidays_security.xml:64]:
```suggestion
        <field name="domain_force">[('employee_id.user_id', '=', user.id), ('state', 'not in', ['validate', 'validate1'])]</field>
```

File: addons/hr_holidays/static/src/scss/time_off.scss [hunk 2]
```diff
@@ -1,9 +1,9 @@
 .o_timeoff_calendar .o_content {
     .o_timeoff_container {
-        height: 6rem;
+        height: 6.0rem;
 
         + .o_calendar_container {
-            height: calc(100% - 6rem);
+        height: calc(100% - 6.0rem);
```
Comment (kba-odoo) [inline addons/hr_holidays/static/src/scss/time_off.scss:6]: Can be reverted, it doesn't change anything

**Review by kbapt — CHANGES_REQUESTED** [review]

**Comment by mart-e** [comment]
@robodoo override=ci/security
Reason: false positifve, changing the description

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**